app/Metadata/Extractor.php

```diff
@@ -170,6 +170,11 @@ public static function createFromFile(NativeLocalFile $file): self
 			}
 		}
 
+		// ! By changing the logic of $reader to always return an Exif object or throw an excepion, this would make this code safer.
```

Comment:
```suggestion
		//  TODO: By changing the logic of $reader to always return an Exif object or throw an excepion, this would make this code safer.
```

Please add `TODO` to these kind of comments. These comments are captured by SonarCloud as a code smell so they won't be lost. Also my IDE looks out for `TODO` in comments and highlights these comments in a special way.

Reply:
Fixed myself in https://github.com/LycheeOrg/Lychee/pull/1336/commits/e5a752740d18961ba0c751a5c8768ec0ff5c5ee5.